pkg/sql/catalog/tabledesc/validate_test.go

```diff
@@ -91,10 +91,11 @@ var validationMap = []struct {
 			"NextMutationID": {
 				status: todoIAmKnowinglyAddingTechDebt,
 				reason: "initial import: TODO(schema): add validation"},
-			"FormatVersion": {status: thisFieldReferencesNoObjects},
-			"State":         {status: thisFieldReferencesNoObjects},
-			"OfflineReason": {status: thisFieldReferencesNoObjects},
-			"Checks":        {status: iSolemnlySwearThisFieldIsValidated},
+			"FormatVersion":     {status: thisFieldReferencesNoObjects},
+			"State":             {status: thisFieldReferencesNoObjects},
+			"OfflineReason":     {status: thisFieldReferencesNoObjects},
+			"LastWriteableTime": {status: thisFieldReferencesNoObjects},
```

Comment:
Can/should this be validated with a "this should not be set unless the table is OFFLINE" check?